sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -289,7 +289,8 @@ def send_message(
         :paramtype chat_message_type: Union[str, ~azure.communication.chat.ChatMessageType]
         :keyword str sender_display_name: The display name of the message sender. This property is used to
             populate sender name for push notifications.
-        :keyword dict[str, str] metadata : Message metadata.
+        :keyword metadata : Message metadata.
```

Comment:
Remove the extra space before the `:`

Reply:
Could you also make this documentation change in the async client?

Reply:
Hi, I removed the extra space and make it back to the same line like other keyword